Makefile

```diff
@@ -1,6 +1,7 @@
 .DEFAULT_GOAL = help
 
-DEV = docker-compose --file docker-compose.dev.yml
```

Comment:
Since we probably won't use more than one container, little point in using Docker Compose.